pulumi/integration_tests/__main__.py

```diff
@@ -1,30 +1,57 @@
+import json
 import os
 import sys
 from pathlib import Path
 
 sys.path.insert(0, "..")
 
-from typing import Optional, cast
+from typing import Mapping, Optional, cast
 
 import pulumi_aws as aws
 import pulumi_nomad as nomad
 from infra import config
 from infra.autotag import register_auto_tags
-from infra.docker_image_tag import version_tag
+from infra.docker_images import DockerImageIdBuilder, version_tag
 from infra.nomad_job import NomadJob, NomadVars
 from infra.quiet_docker_build_output import quiet_docker_output
 
 import pulumi
 
 
+def _container_images(
```

Comment:
The fact that this same logic is repeated in both `__main__.py` files is a further argument for consolidating it.

Reply:
done